tasks/gulp/__tests__/after-build-package.test.js

```diff
@@ -89,4 +98,42 @@ describe('package/', () => {
       await sassRender({ file: allScssFile })
     })
   })
+
+  describe('components', () => {
+    it('should each have macro-options.json that contains JSON', done => {
+      const componentNames = lib.allComponents.slice()
+
+      const getMacroOptionsContents = () => {
+        return componentNames.map(name => {
+          const filePath = path.join(configPaths.package, 'components', name, 'macro-options.json')
+          return readFile(filePath, 'utf8')
+            .then((data) => {
+              var newData = JSON.parse(data)
+              // We expect the component JSON to contain at least "name" and "type"
+              expect(newData).toEqual(
+                expect.arrayContaining([
+                  expect.objectContaining({
+                    name: expect.anything(),
+                    type: expect.anything(),
+                    required: expect.anything(),
```

Comment:
Non-blocking but I think these could be more strictly typed to bool for example in 'required'?

Not a big deal, this test is really nice.

Reply:
Good point, I've made it stricter by checking for strings with `name`, `type` and `description`. For `type`, I don't know if there is a way to check if the value is a boolean? [jest-extended](https://www.npmjs.com/package/jest-extended) has `toBeBoolean()`.

Reply:
does expect.any(Boolean) work?